utils/utils.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package utils
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"time"
+)
+
+func CurrentMS() int64 {
+	return time.Now().UnixNano() / int64(time.Millisecond)
+}
+
+func GoTickerWithRecover(handler func(), tick time.Duration, done context.Context, restartAfterPanic bool, recoverLogger io.Writer) {
```

Comment:
Add comments explain what this function does,

Reply:
done